vue/src/TrackMarketplace/TrackMarketplace.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+/* eslint-disable no-underscore-dangle */
+
+import '../types';
+
+const { $ } = window;
+
+function makeContentBlock(
+  $element: JQuery,
+  pluginName: string,
+  contentPiece: string,
+  target: string,
+) {
+  $element.attr('data-track-content', '');
+  $element.attr('data-content-name', pluginName);
+  $element.attr('data-content-piece', contentPiece);
+  $element.attr('data-content-target', target);
+}
+
+export default {
+  mounted(el: HTMLElement): void {
+    $(el).find('.plugin').each((index, plugin) => {
+      const $plugin = $(plugin);
+      const pluginName = $plugin.find('[piwik-plugin-name]').attr('piwik-plugin-name') || '';
```

Comment:
@diosmosis This will not work since we have updated the file
Refer: https://github.com/matomo-org/matomo/pull/18966/files#diff-db819e82ba4dd2ddc785921f6c3d5834fec8f64bdb154f9ac0d1f31fdecfddc6

Reply:
attribute [piwik-plugin-name] is no longer set

Reply:
actually this is a regression since release of 4.9, its better to fix here